pymilo/transporters/transporter.py

```diff
@@ -56,6 +55,26 @@ def deserialize(self, data, key, model_type):
         :return: pymilo deserialized output of data[key]
         """
 
+    @abstractmethod
+    def bypass(self, content):
+        """
+        Determine whether to bypass transporting on this content or not.
+
+        :param content: either a ML model object's internal data dictionary or an object associated with the json string of a pymilo serialized ML model.
+        :type content: object
+        :return: boolean, whether to bypass or not
+        """
+
+    @abstractmethod
+    def reset(self):
+        """
+        Reset internal data structures of the transport object.
+
+        Some Transporters may be stateful and have internal data structures getting filled during transportation.
```

Comment:
No need for that line, I guess.

Reply:
as I said before, I think it's good to say some transporters might be stateful, because someone may ask why there is a need to have a reset function at all.